openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -518,6 +518,29 @@ def test_program_completion_with_no_id_professional(self, mock_get_certificates_
         meter = ProgramProgressMeter(self.user)
         self.assertEqual(meter.completed_programs, [program['uuid']])
 
+    @ddt.data(
+        ('honor', 'verified', False),
+        ('verified', 'verified', True),
+        ('honor', 'professional', False),
+        ('professional', 'professional', True),
+        ('audit', 'verified', False),
+        ('audit', 'professional', False)
+    )
+    @ddt.unpack
+    @mock.patch(UTILS_MODULE + '.ProgramProgressMeter.completed_course_runs', new_callable=mock.PropertyMock)
+    def test_is_course_complete(self, certificate_type, course_type, is_completed, mock_completed_course_runs,
```

Comment:
Please clean up this test. At a minimum, you should be testing the one case you're trying to fix here (equating credit with verified).